03-terraform-outputs-across-stacks/config.tm.hcl

```diff
@@ -0,0 +1,13 @@
+# generates the backend.tf in each stack
+generate_hcl "backend.tf" {
+  content {
+    terraform {
+      backend "s3" {
+      bucket = "data-sharing-state-bucket"
+      key    = "${terramate.stack.path.relative}/terraform.state"
+      region = "us-west-2"
+      encrypt = true
+     }
+  }
+  }
+}
```

Comment:
run `terramate fmt` to format all Terramate files (indentation is off here) and add new line to the end of the file

Reply:
Ran FMT